piksi_tools/console/update_view.py

```diff
@@ -54,6 +55,46 @@ def parse_version(version):
     return gitversion_parse(version)
 
 
+class NetworkCheck():
+
+    CONNECTION_TIMEOUT = 1.0
+    THREAD_INTERVAL = 5.0
```

Comment:
Can we bump this to 20 or 30 seconds?  A 5 second interval will make the console consume about 2mb a day over a 24hr period of Internet traffic (when it previously only consumed Internet traffic when directed, or when checking the firmware version, afaik).

@denniszollo @switanis Do you think it would be a concern for customers if the console started to ping www.google.com regularly?